README.md

```diff
@@ -62,14 +62,11 @@ We did our best to keep the syntax and overhead as little as possible:
 ```php
 class PostData extends DataTransferObject
 {
-    /** @var string */
-    public $title;
+    public string $title;
     
-    /** @var string */
-    public $body;
+    public string $body;
     
-    /** @var \Author */
-    public $author;
+    public \Author $author;
```

Comment:
I think the examples should "import" the real classes when used here; and maybe this should be documented that this works.

The important thing to remember is that for phpdoc you CANNOT use the imported version, but this restriction doesn't apply to _actual_ types as the type is already correctly resolved by the PHP engine itself when reflections are used.

Reply:
Yeah I am not opposed to that. Looks better and more fluent with the rest of the doc.

Reply:
Latest commit provides an update that resolves this.